src/vs/editor/browser/controller/mouseHandler.ts

```diff
@@ -424,6 +450,36 @@ class MouseDownOperation extends Disposable {
 		// Overwrite the detail of the MouseEvent, as it will be sent out in an event and contributions might rely on it.
 		e.detail = this._mouseState.count;
 
+		if (!this._mouseState.altKey // we don't support multiple mouse
```

Comment:
On top of it, we should enter this case only if the editor has precisely one selection (to avoid the multiple selection case altogether)